provisioning/service/samples/Getting Started/IndividualEnrollmentX509Sample/IndividualEnrollmentX509Sample.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Samples
+{
+    internal class IndividualEnrollmentX509Sample
+    {
+        private const string RegistrationId = "myvalid-registratioid-csharp";
+        private readonly ProvisioningServiceClient _provisioningServiceClient;
+        private readonly X509Certificate2 _issuerCertificate;
+
+        // Optional parameters
+        private const string OptionalDeviceId = "myCSharpDevice";
```

Comment:
Let's make all parameters passable, and not something someone would need to change the code to use.

See: https://github.com/Azure/azure-iot-sdk-csharp/blob/main/iothub/device/samples/Getting%20Started/InvokeDeviceMethod/Parameters.cs
https://github.com/Azure/azure-iot-sdk-csharp/blob/main/iothub/device/samples/Getting%20Started/InvokeDeviceMethod/Program.cs#L24-L27